modules/apps/collaboration/document-library/document-library-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -236,6 +236,7 @@ page import="com.liferay.taglib.search.ResultRow" %><%@
 page import="com.liferay.taglib.util.PortalIncludeUtil" %><%@
 page import="com.liferay.trash.kernel.model.TrashEntry" %>
 
+
```

Comment:
Please fix that...You don't need to have any change on that file.

Reply:
this import is the redundancy code (I have added this line before) because I deleted this class in my code.

Reply:
Ý anh là tại sao em lại commit cái file init.jsp trong khi nó chẳng thay đổi gì cả? Nếu nhìn vào thì em chỉ add 1 empty line ở line 239 thì không có ý nghĩa gì hết. 
Em phải cận thận hơn ở việc commit những file nào cần thiết, và review lại những gì mà mình thay đổi.

Reply:
ak vâng a. em hiểu rồi a.